app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -641,7 +654,7 @@ private void setUpdateIntervalText() {
                 val = String.format(context.getString(R.string.pref_autoUpdateIntervallOrTime_at),
                         timeOfDayStr);
             } else {
-                val = context.getString(R.string.pref_smart_mark_as_played_disabled);
+                val = context.getString(R.string.pref_smart_mark_as_played_disabled);  // TODO: Is this a bug? Otherwise document why is this related to smart mark???
```

Comment:
Again, the issue is with transifex. You cannot tell transifex that you just renamed a string. We push the new strings file and the disabled string has to be translated to each language via their web UI. There is no shortcut for that.